.ado/templates/react-native-init.yml

```diff
@@ -89,6 +89,7 @@ steps:
   - ${{ if eq(parameters.projectType, 'lib') }}:
     - script: |
         npx --yes create-react-native-module@0.20.2 --package-name "testcli" testcli
+        rd /s /q testcli\android
```

Comment:
This is necessary because this old `create-react-native-module` module now no longer creates working android code (as of this integrate), which prevents `react-native config` from running successfully, which breaks run-windows.